libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -345,6 +343,53 @@ class MapboxTripNotification constructor(
                 STEP_MANEUVER_MODIFIER_UTURN == maneuverModifier)
     }
 
+    private fun setFreeDriveMode(isFreeDriveMode: Boolean) {
+        updateEstimationBlockVisibility(isFreeDriveMode)
+        updateFreeDriveTextVisibility(isFreeDriveMode)
+        updateManeuverImageResource(R.drawable.ic_maneuver_turn_0)
+        updateEndNavigationBtnText(isFreeDriveMode)
+    }
+
+    private fun updateEstimationBlockVisibility(isFreeDriveMode: Boolean) {
+        collapsedNotificationRemoteViews?.setViewVisibility(
+            R.id.estimationBlock,
+            if (isFreeDriveMode) GONE else VISIBLE
+        )
+        expandedNotificationRemoteViews?.setViewVisibility(
+            R.id.estimationBlock,
+            if (isFreeDriveMode) GONE else VISIBLE
+        )
+    }
+
+    private fun updateFreeDriveTextVisibility(isFreeDriveMode: Boolean) {
+        collapsedNotificationRemoteViews?.setViewVisibility(
+            R.id.freeDriveText,
+            if (isFreeDriveMode) VISIBLE else GONE
+        )
+        expandedNotificationRemoteViews?.setViewVisibility(
+            R.id.freeDriveText,
+            if (isFreeDriveMode) VISIBLE else GONE
+        )
+    }
+
+    private fun updateManeuverImageResource(@DrawableRes maneuverResId: Int) {
```

Comment:
FYI that the PR #2481 has just been merged, please refer to the latest code: https://github.com/mapbox/mapbox-navigation-android/blob/master/libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt#L308
```kotlin
when (Build.VERSION.SDK_INT > Build.VERSION_CODES.M)
```
we need to update the icon or bitmap base on the different OS accordingly.